sdk/servicefabricmanagedclusters/azure-mgmt-servicefabricmanagedclusters/CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Release History
 
+## 2.1.0b5 (2025-11-10)
+
+### Features Added
+
+  - Model `ServiceFabricManagedClustersManagementClient` added parameter `cloud_setting` in method `__init__`
+  - Added model `ApplicationFetchHealthRequest`
+  - Added enum `HealthFilter`
+  - Added model `RestartDeployedCodePackageRequest`
+  - Added enum `RestartKind`
+  - Added model `RestartReplicaRequest`
+  - Added model `RuntimeApplicationHealthPolicy`
+  - Added enum `RuntimeFailureAction`
+  - Added enum `RuntimeRollingUpgradeMode`
+  - Added model `RuntimeRollingUpgradeUpdateMonitoringPolicy`
+  - Added model `RuntimeServiceTypeHealthPolicy`
+  - Added model `RuntimeUpdateApplicationUpgradeParameters`
+  - Added enum `RuntimeUpgradeKind`
+  - Model `ApplicationsOperations` added method `begin_fetch_health`
+  - Model `ApplicationsOperations` added method `begin_restart_deployed_code_package`
+  - Model `ApplicationsOperations` added method `begin_update`
```

Comment:
@ChenxiJiang333 You already declared this change in "breaking change" part so no need to keep it here.